server/routes/index.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const router = express();
+
+router.get('/', (req, res) => {
+  return res.status(200).json({
+    status: res.statusCode,
+    message: `Hello there! This is Author's haven`
```

Comment:
Good work @chuxmykel, but I think for the message property since you are not outputting any variable, it's best you use single quotes rather than template literal string to avoid eslint error

Reply:
Well in this case single quote won't work, the template string is fine or you use double quotes in that case... In the advent that string has single quotes like apostrophes within it, the string can be wrapped in double quote.

Reply:
I will use the single quotes and maybe escape the `'` character. Let me get to it immediately. Thanks guys